pkg/gui/gui.go

```diff
@@ -619,6 +619,25 @@ func (gui *Gui) goEvery(interval time.Duration, function func() error) {
 	}()
 }
 
+func (gui *Gui) startBackgroundFetch() error {
+	g := gui.g
```

Comment:
Is there a reason that you use g instead of gui.g?

Reply:
err because i made the code work first and forgot about this second? :) fixed

Reply:
Haha okay, i thought so, then i have no other comments, we just have to wait for jesse to review